cg_lims/set/udfs.py

```diff
@@ -8,6 +8,8 @@
 
 LOG = logging.getLogger(__name__)
 
+READS_FIELD = "# Reads"
+Q30_FIELD = "% Bases >=Q30"
```

Comment:
```suggestion
class UdfFields(strEnum)
READS_FIELD = "# Reads"
Q30_FIELD = "% Bases >=Q30"
```

are these used for udfs only? Maybe we can call them UdfConstants

Reply:
Yeah, I think we should use an enum for these 👍